src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -379,6 +379,17 @@ body.page-tribe-attendee-registration {
 		right: 0;
 		top: 0;
 	}
+
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Twenty twenty
+	 * ------------------------------------------------------------------------- */
+
+	&.tribe-theme-twentytwenty {
+
+		.entry-content .tribe-tickets__registration {
+			max-width: 960px !important;
```

Comment:
We really can't do it with a combo of ID + Classes + Elements? If not it's ok.

Reply:
No, because they rely on the `:not()` and it's taking over.